test/integration/targets/nxos_ntp_auth/tasks/nxapi.yaml

```diff
@@ -18,21 +20,8 @@
 - name: set test_items
   set_fact: test_items="{{ test_cases.files | map(attribute='path') | list }}"
 
-- name: enable nxapi
```

Comment:
Where is this done now?

Reply:
Where it always was, in `prepare_nxos_tasks`. Specifying it here is redundant

Reply:
ah, yes, thanks